e2e/testcases/death/declaration/death-declaration-9.spec.ts

```diff
@@ -0,0 +1,508 @@
+import { test, expect, type Page } from '@playwright/test'
+import { createPIN, goToSection, login } from '../../../helpers'
+import faker from '@faker-js/faker'
+
+test.describe.serial('9. Death declaration case - 9', () => {
+  let page: Page
+  const require = 'Required for registration'
```

Comment:
```suggestion
  const REQUIRED = 'Required for registration'
```
Just to emphasize this a bit